src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,3 +1,13 @@
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
+import './UserInfo.scss';
+
+import { User } from '../../types/User';
+
+type Props = {
+  user: User
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <a className="UserInfo" href={user.email}>
```

Comment:
```suggestion
  <a className="UserInfo" href={`mailto:${user.email}`}>
```
Your mail link for User doesn't work correctly and tests failed